source/build/Android/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.0'
+        classpath 'com.android.tools.build:gradle:3.5.2'
```

Comment:
Feel free to won't fix. Version numbers seem like they could be inputs through a .ejs file. 

But it effectively would just move this number into another place unless we could make a call to get the latest version of gradle (or whatever is currently on the machine). 

While looking for Gradle environment variables I ran across: https://docs.gradle.org/current/userguide/single_versions.html which said we could possibly declare a version number range instead. BUT the very bottom of the page mentions we can declare a dependency constraint: 

A recommended practice for larger projects is to declare dependencies without versions and use dependency constraints for version declaration. The advantage is that dependency constraints allow you to manage versions of all dependencies, including transitive ones, in one place.

Reply:
While I agree with you on principle, the fact of the matter is that it's already a fixed version, and my Android Studio just updated the project somewhat against my will.
Rolling it back may not be safe, given the other changes.
And the rest of what you say is out-of-scope for this bugfix.